src/ui/common/src/components/tables/virtualizedTable.tsx

```diff
@@ -0,0 +1,252 @@
+import { Box, Typography } from '@mui/material';
+import { styled, Theme } from '@mui/material/styles';
+import TableCell from '@mui/material/TableCell';
+import clsx from 'clsx';
+import * as React from 'react';
+import {
+  AutoSizer,
+  Column,
+  Table,
+  TableCellRenderer,
+  TableHeaderProps,
+} from 'react-virtualized';
+
+const classes = {
```

Comment:
Are these not exposed as constants from `react-virtualized`? I know MUI does this. It'd make it more maintainable for the future (i.e., if these class names change).

Reply:
I don't think so after poking around a bit. The example code on the web also have these class name strings hard-coded..